webplugin/js/app/conversation/km-conversation-service.js

```diff
@@ -20,11 +20,18 @@ Kommunicate.conversation = {
             );
             return;
         }
+
         var conversationDetail = data && data.groupFeeds[0];
+        const isConversationRated = kmLocalStorage.getItemFromLocalStorage(
+            conversationDetail.id
+        );
```

Comment:
instead of localStorage use the session storage

Reply:
If we use session storage instead of local storage, there will be one issue: once the user closes the CSAT feedback, it will not appear again until the page is refreshed or reopened. When the user closes and reopens the page, the CSAT feedback will reappear once, as session storage is cleared at the end of each session. so that's why i am using the local storage instead of session storage.